src/main/java/org/mitre/inferno/FHIRPathEvaluator.java

```diff
@@ -0,0 +1,57 @@
+package org.mitre.inferno;
+
+import com.google.gson.Gson;
+import java.io.IOException;
+import java.util.List;
+import java.util.stream.Collectors;
+import org.hl7.fhir.exceptions.FHIRException;
+import org.hl7.fhir.r5.context.SimpleWorkerContext;
+import org.hl7.fhir.r5.formats.JsonParser;
+import org.hl7.fhir.r5.model.Base;
+import org.hl7.fhir.r5.model.DataType;
+import org.hl7.fhir.r5.model.Resource;
+import org.hl7.fhir.r5.utils.FHIRPathEngine;
+
+public class FHIRPathEvaluator extends FHIRPathEngine {
+
+  public FHIRPathEvaluator() throws IOException {
+    super(new SimpleWorkerContext());
+  }
+
+  @Override
+  public String evaluateToString(Base base, String path) {
+    List<Base> result = this.evaluate(base, path);
+    return '['
+        + result.stream().map(this::baseToJson).collect(Collectors.joining(","))
+        + ']';
+  }
+
+  private String baseToJson(Base item) {
+    return String.format("{\"type\":\"%s\",\"value\":%s}", item.fhirType(), baseToString(item));
+  }
+
+  String baseToString(Base item) {
+    String type = item.fhirType();
+    if (item.isPrimitive()) {
+      String value = item.primitiveValue();
+      switch (type) {
+        case "boolean":
+        case "integer":
+        case "decimal":
+          return value;
+        default: // TODO: handle date, dateTime, etc. appropriately
```

Comment:
Does this todo mean that date/datetime will be returned incorrectly? If so, is that a prerequisite to being able to use this evaluator for Inferno work?

Reply:
In this comment, I was noting that I didn't handle all of the JSON primitives properly, but I just pushed a change to fix this. Thanks for reminding me! 😅  I just discovered the column that describes the proper JSON representations of each primitive on the [FHIR Datatypes page](https://www.hl7.org/fhir/datatypes.html)